homeassistant/components/sensor/enphase_envoy.py

```diff
@@ -32,20 +32,20 @@
 ICON = 'mdi:flash'
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_IP_ADDRESS): cv.string,
+    vol.Optional(CONF_IP_ADDRESS, default="envoy"): cv.string,
```

Comment:
Make `"envoy"` a constant and reference that here. Additionally, because of this, I assume documentation changes re in order?